tests/psi4numpy/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 add_subdirectory(cphf)
 add_subdirectory(dfmp2)
 add_subdirectory(rhf)
+add_subdirectory(rhf_gradient)
```

Comment:
underscore to dash here, too, I think

Reply:
Sorry! Too late to be working, it seems :)